src/HexagonAlignLoads.h

```diff
@@ -0,0 +1,22 @@
+#ifndef HALIDE_HEXAGON_ALIGN_LOADS_H
+#define HALIDE_HEXAGON_ALIGN_LOADS_H
```

Comment:
Yeah, I'm not saying we should use it anywhere but Hexagon. I'm not commenting here on anything beyond just the naming of it.